src/adhocracy_frontend/adhocracy_frontend/static/js/ResourcesBase.ts

```diff
@@ -3,9 +3,7 @@ export interface ISheetMetaApi {
     readable : string[];
     editable : string[];
     creatable : string[];
-    /* tslint:disable:variable-name */
```

Comment:
Are these changes related to 6620eeac45ea3f0c299643e8df20a4824f3c45d7 (mv `isInstanceOf`to ResourceUtil)?